kb/eekb/agent_of_finding_sc_elements_without_any_main_identifier/agent_component/lib_component_agent_of_finding_sc_elements_without_any_main_identifier/sc_agent_of_finding_sc_elements_without_any_main_identifier.scs

```diff
@@ -0,0 +1,49 @@
+sc_agent_of_finding_sc_elements_without_any_main_identifier
+=> nrel_main_idtf:
+	[sc-агент поиска sc-элементов, имеющих системный идентификатор, но не имеющих ни одного основного] (* <- lang_ru;; *);
+	[sc-agent of finding sc-elemnts without any main identifier] (* <- lang_en;; *);
```

Comment:
sc-agent of finding sc-elements with system identifier but without any main identifier